.github/workflows/build.yml

```diff
@@ -230,7 +230,7 @@ jobs:
     - name: Run Terraform Tests
       run: |
         cd moto-terraform-tests
-        bin/run-tests -i ../tests/terraform-tests.success.txt -e ../tests/terraform-tests.failures.txt
+        AWS_DEFAULT_REGION=us-east-1 AWS_ALTERNATE_REGION=eu-west-1 bin/run-tests -i ../tests/terraform-tests.success.txt -e ../tests/terraform-tests.failures.txt
```

Comment:
Hey @bblommers @whummer, We have added these (`AWS_DEFAULT_REGION` and `AWS_ALTERNATE_REGION`) parameters in the run command as they are required in the terraform tests.